src/ripple/protocol/SecretKey.h

```diff
@@ -72,25 +75,25 @@ class SecretKey
     const_iterator
     begin() const noexcept
     {
-        return buf_;
+        return buf_.data();
```

Comment:
Why not `buf_.cbegin();`?

Reply:
Will do.

Reply:
Addressed in e71cfc80b1583e5055c9f4ed5828b2e7c7deeeb4